app/services/proofing/base.rb

```diff
@@ -25,7 +25,7 @@ def optional_attributes(*optional_attributes)
       end
 
       def attributes
-        required_attributes.concat optional_attributes
+        [*required_attributes, *optional_attributes]
```

Comment:
`.concat` will mutate, so there was a lot of duplication in specs and they were inconsistent.

This changes this to make a fresh copy each call